### PR TITLE
Postgres functions now roundtrip correctly into a schema.rb file

### PR DESCRIPTION
**Pull request by wwahammy**
The bug we found related to roundtripping SQL functions with `\` characters in them was corrected in [fx](https://github.com/teoljungberg/fx/commit/a9c262836c02dabced6c18172f022df2efff6b5b) but that version hasn't been released yet. Therefore, we're using the latest commit for fx that is available at this point from the official repo.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -461,8 +461,6 @@
     t.datetime "updated_at",                                 null: false
   end
 
-  add_index "fee_coverage_detail_bases", ["fee_era_id"], name: "index_fee_coverage_detail_bases_on_fee_era_id", using: :btree
```
Comment (clarissalimab) [inline db/schema.rb]: What happened to those lines?

Comment (wwahammy) [reply db/schema.rb]: hmmmm, something that should not have happened

Comment (wwahammy) [reply db/schema.rb]: It's corrected now, please review!

File: db/schema.rb [hunk 2]
```diff
@@ -1251,13 +1251,13 @@
...
   SQL
-  create_function :update_phone_index_on_supporters, sql_definition: <<-SQL
+  create_function :update_phone_index_on_supporters, sql_definition: <<-'SQL'
       CREATE OR REPLACE FUNCTION public.update_phone_index_on_supporters()
        RETURNS trigger
        LANGUAGE plpgsql
       AS $function$
                   BEGIN
-                    new.phone_index = (regexp_replace(new.phone, '\\D','', 'g'));
+                    new.phone_index = (regexp_replace(new.phone, '\D','', 'g'));
```
Comment (clarissalimab) [inline db/schema.rb]: Oops :cry:

Comment (wwahammy) [reply db/schema.rb]: That's correct now! It was always supposed to be a single `\` but before it was stripping out the first slash.

**Comment by wwahammy** [comment]
I'm going to mergy merge.

**Pull request merged**